tests/common/quantization/test_quantizer_propagation_solver.py

```diff
@@ -840,8 +840,7 @@ def __init__(self, quantization_trait, config, op_meta=UnknownMetatype):
     BRANCH_TRANSITION_TEST_CASES = [
         # Downward branches are quantization-agnostic
         BranchTransitionTestStruct(
-            init_node_to_trait_and_configs_dict=
-            {
+            init_node_to_trait_and_configs_dict={
```

Comment:
New rule conflicts with black formatter. 
This block under `# fmt: skip` comments, that disable formatting.
I dont think that we need check formatting by several tools.

Reply:
Discussed offline% decided to ignore the foramting rules intersected with black: 
[E251](https://docs.astral.sh/ruff/rules/unexpected-spaces-around-keyword-parameter-equals/) should be covered by black already
[E201](https://docs.astral.sh/ruff/rules/whitespace-after-open-bracket/) should be covered by black already
[E231](https://docs.astral.sh/ruff/rules/missing-whitespace/) should be covered by black already
[E203](https://docs.astral.sh/ruff/rules/whitespace-before-punctuation/) should be covered by black already